test/ai.ts

```diff
@@ -0,0 +1,27 @@
+import { before, describe, it } from 'mocha';
+import { fetchUrlSummary } from '../bots/ai.js';
+import { ReadsConfig } from '../bots/reads.js';
+import { expect } from 'chai';
+
+describe(('Fetching URL Summary'), () => {
+    let config: ReadsConfig
+    before(() => {
+        config = {
+            botToken: '',
+            openaiApi: process.env.OPENAI_API_KEY,
```

Comment:
since these are unit tests, we don't want them to actually call openai. they just need to exercise the various code paths of the function. that is, the `try/catch` cases and the `if` cases.

you can use mocking to bypass calls to the real openai server and also verify that the calls to openai are made in the tests.